torch/csrc/distributed/rpc/process_group_agent.h

```diff
@@ -138,8 +138,15 @@ class ProcessGroupAgent : public RpcAgent {
   void handleSend(const SendWork& work);
   // put RecvWork into a queue and notify the worker thread
   void enqueueRecv(RecvWork work);
-  // receiving messages
+  // Loop for receiving messages. Calls listenLoopInternal and handles errors
+  // such as timeouts on the process group.
+  virtual void listenLoopInternal();
```

Comment:
Does this function need to be virtual? Seems like the only child classes of ProcessGroupAgent are those in part of tests and don't override listen loop.

Reply:
Right, initially I had some testing that overrides it but I removed that. This PR is already landing though so I'll fix it in a followup